TW-939  Doc Create collection when starting proxy capture session (#4574)
postmanlabs/postman-docs: src/pages/docs/sending-requests/capturing-request-data/capturing-http-requests.md

    > You can also capture cookies and sync them to Postman without starting a debug session. Learn more about [syncing cookies](/docs/sending-requests/capturing-request-data/syncing-cookies/).

> 1. Requests will be saved in a debug session in the **History** tab in the sidebar. Select **Save requests to a collection** and select a collection from the list to also save the requests there. You can also type the name of a new collection and select the Create action to create a new collection.

Reviewer: ```suggestion
1. Requests will be saved in a debug session in the **History** tab in the sidebar. Select **Save requests to a collection** and select a collection from the list to also save the requests there. To create a new collection in which to save requests, enter a name for the new collection and select the Create action.
```
